sinri/HebrewStrongDictionary.html

```diff
@@ -3,22 +3,11 @@
 <head>
 	<meta charset="utf-8">
 	<title>Hebrew Strong Dictionary</title>
-	<script src="//cdn.bootcss.com/jquery/3.2.0/jquery.min.js"></script>
```

Comment:
You shouldn't remove this since jquery does not exist in this repo.  This line does appear to be missing the "http:", however.